build.gradle.kts

```diff
@@ -14,4 +14,5 @@ plugins {
     alias(libs.plugins.kotlinAndroid) apply false
     alias(libs.plugins.androidLibrary) apply false
     alias(libs.plugins.hilt) apply false
+    id("com.google.devtools.ksp") version "1.9.20-1.0.14" apply false
```

Comment:
can you follow the pattern for other plugins - https://github.com/amazon-connect/amazon-connect-chat-android/blob/main/gradle/libs.versions.toml